reactioncommerce/reaction
# refactor: use new URL instead of url.parse

Resolves #5704
Impact: **minor**  
Type: **refactor**

Change all url.parse to new URL(). See https://nodejs.org/api/url.html#url_url

## Breaking changes
None


## Testing
1. See that the api starts
1. Try the `primaryShopId` query, which uses `new URL`, see if it returns the ID
